### PR TITLE
refactor(front_vehicle_velocity_estimator): rework parameters

### PR DESCRIPTION
**Pull request by YuqiHuai**
## Description

ROS node configuration refactored according to [Guideline](https://github.com/orgs/autowarefoundation/discussions/3371).

## Related links

None.

## Tests performed

1. Build `colcon build --symlink-install --cmake-args -DCMAKE_BUILD_TYPE=Release --packages-up-to front_vehicle_velocity_estimator`
2. Launch `ros2 launch front_vehicle_velocity_estimator front_vehicle_velocity_estimator.launch.xml`

## Notes for reviewers

None.

## Interface changes

1. Update CMakeLists.txt to include `config`
2. Update README.md
3. Create param file and schema file
4. Update launch file
5. Update calls to `declare_parameter`

## Effects on system behavior

None.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: perception/front_vehicle_velocity_estimator/README.md [hunk 1]
```diff
@@ -28,18 +28,6 @@ This package can:
...
-| `threshold_pointcloud_z_low`  | float  | The threshold for z position value of point when choosing nearest neighbor point within front vehicle [m]. If z < `threshold_pointcloud_z_low`, the point is considered to noise like ground.                                                                               | 0.6f          |
-| `threshold_relative_velocity` | double | The threshold for min and max of estimated relative velocity ($v_{re}$) [m/s]. If $v_{re}$ < - `threshold_relative_velocity` , then $v_{re}$ = - `threshold_relative_velocity`. If $v_{re}$ > `threshold_relative_velocity`, then $v_{re}$ = `threshold_relative_velocity`. | 10.0          |
-| `threshold_absolute_velocity` | double | The threshold for max of estimated absolute velocity ($v_{ae}$) [m/s]. If $v_{ae}$ > `threshold_absolute_velocity`, then $v_{ae}$ = `threshold_absolute_velocity`.                                                                                                          | 20.0          |
+{{ json_to_markdown("perception/front_vehicle_velocity_estimator/schema/front_vehicle_velocity_estimator.schema.json") }}
```
Comment (scepter914) [inline perception/front_vehicle_velocity_estimator/README.md:33]: [MUST]
This change is very inconvenience because we cannot see parameters description at github.
I think the parameter description should be written by markdown lile [this PR](https://github.com/autowarefoundation/autoware.universe/pull/4871/files).

**Review by scepter914 — CHANGES_REQUESTED** [review]

File: perception/front_vehicle_velocity_estimator/config/front_vehicle_velocity_estimator.param.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 UCI SORA LAB.
```
Comment (scepter914) [inline perception/front_vehicle_velocity_estimator/config/front_vehicle_velocity_estimator.param.yaml:1]: [MUST]
Delete copyright as discussed at https://github.com/autowarefoundation/autoware.universe/pull/4695

Comment (YuqiHuai) [reply perception/front_vehicle_velocity_estimator/config/front_vehicle_velocity_estimator.param.yaml:1]: Got it. Following the guideline

> We don't add copyright notice to parameter files (.param.yaml) or schema files (.schema.json)

copyright notice should be removed from param file.

File: perception/front_vehicle_velocity_estimator/README.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ This package can:
 | `~/output/objects`                    | autoware_auto_perception_msgs/msg/DetectedObjects.msg | 3D detected object with twist.                |
 | `~/debug/nearest_neighbor_pointcloud` | sensor_msgs/msg/PointCloud2.msg                       | The pointcloud msg of nearest neighbor point. |
 
-## Node parameter
+## Node Parameter
```
Comment (scepter914) [inline perception/front_vehicle_velocity_estimator/README.md:31]: [Must]
This change is not necessary, so could you revert this change? 
There is lack of uniformity in uppercase and lowercase between "Node Parameter" and "Core parameter".

**Review by scepter914 — CHANGES_REQUESTED** [review]

File: perception/front_vehicle_velocity_estimator/README.md (hunk 3 above)
Comment (kminoda) [reply perception/front_vehicle_velocity_estimator/README.md]: @YuqiHuai Any updates?

**Comment by scepter914** [comment]
Because this PR has stale label and [front_vehicle_velocity_estimator will be deleted](https://github.com/autowarefoundation/autoware.universe/pull/6258), so I close this PR.
Thank you for contributing.
